pkg/virt-launcher/virtwrap/network/podnic_test.go

```diff
@@ -98,21 +100,31 @@ var _ = Describe("podNIC", func() {
 		})
 	})
 	Context("when DHCP startup fails", func() {
+		var (
+			tmpDir string
+			podnic *podNIC
+			domain *api.Domain
+		)
 		BeforeEach(func() {
+			var err error
+			tmpDir, err = ioutil.TempDir(os.TempDir(), "dhcp")
+			Expect(err).ToNot(HaveOccurred())
 			mockNetwork.EXPECT().StartDHCP(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).Return(errors.New("podnic_test: forcing failure at DHCP start"))
+			domain = NewDomainWithBridgeInterface()
+			vmi := newVMIBridgeInterface("testnamespace", "testVmName")
+			api.NewDefaulter(runtime.GOARCH).SetObjectDefaults_Domain(domain)
+			podnic, err = newPodNICWithMocks(vmi)
+			Expect(err).ToNot(HaveOccurred())
+			podnic.dhcpConfigurator.DHCPStartedDirectory = tmpDir
```

Comment:
I a mock was used for `dhcpConfigurator` the next three lines was not needed. We could just mock that `dhcpConfigurator.ImportConfiguration` returns an error.

Reply:
Done